app/routes/thank-you/route.tsx

```diff
@@ -1,3 +1,31 @@
+import { useLoaderData, json } from '@remix-run/react';
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { getEcomApi } from '~/api/ecom-api';
+import { CategoryLink } from '~/components/category-link/category-link';
+import { OrderSummary } from '~/components/order-summary/order-summary';
+import styles from './thank-you.module.scss';
+
+export const loader = async ({ request }: LoaderFunctionArgs) => {
+    const orderId = new URL(request.url).searchParams.get('orderId');
+    if (!orderId) throw new Error('Order not found');
+    const order = await getEcomApi().getOrder(orderId);
+    return json({ order });
+};
+
 export default function ThankYouPage() {
-    return <h1>Thank you</h1>;
+    const { order } = useLoaderData<typeof loader>();
+
+    return (
+        <div className={styles.root}>
+            <h1 className="heading4">Thank You!</h1>
+            <div className={styles.subtitle}>{`You'll receive a confirmation email soon.`}</div>
```

Comment:
why in backticks?

Reply:
Because of `'`

Reply:
Doesn't double quotes work? 
"You'll receive a confirmation email soon."

Is the difference in the Codux: text node in "" and text node based on {} ?

Reply:
Double quotes no.
There is no big diff in Codux when text node is wrapped in {``} or not, only in the props panel it shows `Edit code` instead of text input.
I'll escape it with \&apos;